build.gradle

```diff
@@ -278,14 +278,7 @@ task writeProperties(type: WriteProperties) {
     property("driver.major.version", MAJOR_VERSION)
     property("driver.minor.version", MINOR_VERSION)
     property("driver.full.version", project.version)
-}
-
-/**
- * Write default application name to properties file under common.
- */
-task writeCommonProperties(type: WriteProperties) {
```

Comment:
Don't need common.properties - because we coalesced the projects into one project. The project.properties is generated and so we don't need to track is as source.